tests/simple.rs

```diff
@@ -152,6 +152,16 @@ fn capture_screenshot_jpeg() -> Result<(), failure::Error> {
     Ok(())
 }
 
+#[test]
+fn test_print_file_to_pdf() -> Result<(), failure::Error> {
+    logging::enable_logging();
+    let (_, browser, tab) = dumb_server(include_str!("./pdfassets/index.html"));
+    let local_pdf = tab.wait_until_navigated()?
+                       .print_to_pdf(None)?;
+    assert_eq!(true, local_pdf.len() > 1000);  // an arbitrary size
```

Comment:
This should read `assert!(!local_pdf.is_empty())`. PDFs start with `b"%PDF"`, which would be even better and self-documenting.

Reply:
@lukaslueg a pdf of an empty page can render to > 0 sized.  1000 ensures this isn't an empty pdf

Reply:
I'm not sure if I understand your point. If the test ensures that the buffer is not empty, it should read `assert!(!local_pdf.is_empty())` and avoid arbitrary values. An even better test would be to ensure that the PDF magic bytes are there, something to the tune of `assert!(local_pdf.starts_with(b"PDF%"))`

Reply:
The pdf magic bytes is a good idea.  I've added it.  The non-zero size test you are recommending does not consider empty pdf generation.  To simulate what I am referring to:

1. cargo build --example print_to_pdf
2. copy the print_to_pdf from the target directory to /tests/pdfassets
3. start a google-chrome-stable headless instance, copy the ws url provided
4. change the working dir to /tests/pdfassts and call " ./print_to_pdf {ws_url} index.html"

This creates a rust.pdf that is 894 bytes but has no content -- it is empty.  local_pdf is 894 bytes yet has no content.